common4j/src/main/com/microsoft/identity/common/java/commands/SilentTokenCommand.java

```diff
@@ -105,26 +105,28 @@ public AcquireTokenResult execute() throws Exception {
                         span.setStatus(StatusCode.OK);
                         return result;
                     }
-                } catch (UiRequiredException | ClientException e) {
-                    if (e.getErrorCode().equals(OAuth2ErrorCode.INVALID_GRANT) // was invalid_grant
-                            && controllers.size() > ii + 1) { // isn't the last controller we can try
-                        exceptionFromFirstController = exceptionFromFirstController == null ? e : exceptionFromFirstController;
-                        continue;
-                    } else if ((e.getErrorCode().equals(ErrorStrings.NO_TOKENS_FOUND)
-                            || e.getErrorCode().equals(ErrorStrings.NO_ACCOUNT_FOUND))
-                            && controllers.size() > ii + 1) {
-                        exceptionFromFirstController = exceptionFromFirstController == null ? e : exceptionFromFirstController;
-                        //if no token or account for this silent call, we should continue to the next silent call.
-                        continue;
```

Comment:
Seems like you removed the `continue` statement but this needs to be put back

Reply:
Continue was redundant.